variables.tf

```diff
@@ -108,6 +108,7 @@ variable "primary_instance" {
     gce_zone          = optional(string)
     availability_type = optional(string)
     machine_cpu_count = optional(number, 2),
+    ssl_mode          = optional(string, "ALLOW_UNENCRYPTED_AND_ENCRYPTED"),
```

Comment:
@rajc242 can you plz change this to

ssl_mode = optional(string)
require_connectors = optional(string)

Reply:
Fixed. PLTL.